common/authen.js

```diff
@@ -200,6 +249,38 @@
                 return _session;
             },
 
+            promptUserPreviousSession: function() {
+                // if there's no stored data OR
+                // there's stored data but we are after it's expiration
+                if (!promptTokenExists() || promptTokenHasExpired()) {
```

Comment:
This means that we're going to show this prompt to the first time users (users that have never logged in before) too. We want to show this only if you have logged in before and your session is expired.